vite.config.js

```
@@ -1,20 +1,50 @@
+            if (assetInfo.name && assetInfo.name.endsWith('.css')) {
+              return 'css/style.css';
+            }
+            return 'assets/[name].[hash][extname]';
+          },
+        },
+        {
+          dir: 'themes/modern-theme/account/resources',
+          format: 'es', // Keep format consistent or adjust as needed
+          entryFileNames: 'js/main.js',
+          assetFileNames: (assetInfo) => {
+            if (assetInfo.name && assetInfo.name.endsWith('.css')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)